models/domains_tlds.php

```diff
@@ -379,29 +379,33 @@ public function edit($tld, array $vars)
                     ? $vars['email_content']
                     : (isset($package->email_content) ? $package->email_content : null),
                 'pricing' => $package->pricing,
-                'option_groups' => (array) (
-                    isset($package->option_groups)
-                        ? $this->Form->collapseObjectArray($package->option_groups, 'id', 'id')
-                        : []
+                'option_groups' => (array)(
+                isset($package->option_groups)
+                    ? $this->Form->collapseObjectArray($package->option_groups, 'id', 'id')
+                    : []
                 ),
-                'meta' => (array) (
-                    isset($vars['meta'])
-                        ? array_merge((isset($package->meta) ? (array) $package->meta : []), $vars['meta'])
-                        : (isset($package->meta) ? $package->meta : [])
+                'meta' => (array)(
+                isset($vars['meta'])
+                    ? array_merge((isset($package->meta) ? (array)$package->meta : []), $vars['meta'])
+                    : (isset($package->meta) ? $package->meta : [])
                 )
             ];
             $fields = json_decode(json_encode($fields), true);
 
             foreach ($fields as $key => $value) {
-                if (is_null($value)) {
+                if (empty($value)) {
```

Comment:
Why the choice to use empty? By defaulting taxable to zero here you may actually be disabling tax when the intent was not to alter it by not submitting that field.